cli/tests/unit/test_error_handler.py

```diff
@@ -35,35 +50,38 @@ def test_send_nominal(error_handler, mocker: MockerFixture) -> None:
     exit_code = error_handler.send(FATAL_EXIT_CODE)
 
     expected_payload = {
-        "headers": {"User-Agent": expected_user_agent},
         "method": "get",
         "url": "https://semgrep.dev/api/agent/deployments/current",
     }
 
+    expected_headers = {
```

Comment:
Are these the headers as SQS sees them or as they are in the body of the requet?

Reply:
These are actual headers that will be forwarded to the SQS.